content/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-neovim.md

```diff
@@ -17,67 +17,60 @@ If you use a Neovim, you can view and incorporate suggestions from {% data varia
+
+- To use {% data variables.product.prodname_copilot %} in Neovim you must have Neovim and Node.js version 17 or below installed. For more information, see the [Neovim documentation](https://neovim.io/doc/) and the [Node.js website](https://nodejs.org/en/).
 
 ## Installing the Neovim extension
 
 {% mac %}
 
 {% data reusables.copilot.install-copilot-in-neovim %}
-   - To install the {% data variables.product.prodname_copilot %} plugin directly, you must know where Neovim stores plugins. To install the plugin, enter the following command in Terminal.
+   - To install {% data variables.product.prodname_copilot %} with Neovim's built-in plugin manager, enter the following command in Terminal.
 
-     ```
-     git clone https://github.com/github/copilot.vim \
-        PATH/TO/NEOVIM/PLUGINS/copilot.vim
-     ```
-1. To configure {% data variables.product.prodname_copilot %}, open Neovim and enter the following command.
+         mkdir -p ~/.config/nvim/pack/github/start
+         git clone https://github.com/github/copilot.vim \
+            ~/.config/nvim/pack/github/start/copilot.vim
 
-   ```
-   :Copilot setup
-   ```
+{% data reusables.copilot.config-enable-copilot-in-neovim %}
 
 {% endmac %}
 
 
 {% windows %}
 
 {% data reusables.copilot.install-copilot-in-neovim %}
-    - To install the {% data variables.product.prodname_copilot %} plugin directly, you must know where Neovim stores plugins. To install the plugin, enter the following command in Git Bash.
+   - To install {% data variables.product.prodname_copilot %} with Neovim's built-in plugin manager, enter the following command in Git Bash.
 
-      ```
-      git clone https://github.com/github/copilot.vim \
-      ~/.config/PATH/TO/YOUR/NEOVIM/CONFIG/FILE/copilot.vim
-1. To configure {% data variables.product.prodname_copilot %}, open Neovim and enter the following command.
+         mkdir -p ~/.config/nvim/pack/github/start
+         git clone https://github.com/github/copilot.vim \
+            ~/.config/nvim/pack/github/start/copilot.vim
 
-   ```
-   :Copilot setup
-   ```
+{% data reusables.copilot.config-enable-copilot-in-neovim %}
 
 {% endwindows %}
 
 
 {% linux %}
 
 {% data reusables.copilot.install-copilot-in-neovim %}
-    - To install the {% data variables.product.prodname_copilot %} plugin directly, you must know where Neovim stores plugins. To install the plugin, enter the following command in Git Bash.
+   - To install {% data variables.product.prodname_copilot %} with Neovim's built-in plugin manager, enter the following command:
+
+         mkdir -p ~/.config/nvim/pack/github/start
+         git clone https://github.com/github/copilot.vim \
+            ~/.config/nvim/pack/github/start/copilot.vim
```

Comment:
The correct path on Windows is `~/AppData/Local/nvim/pack/github/start/copilot.vim`. Unfortunately it's not quite a drop-in replacement, as `~` and `\` are specific to UNIX shells. In the copilot.vim README, I'm currently suggesting the following PowerShell command:

```
          git clone https://github.com/github/copilot.vim.git `
            $HOME/AppData/Local/nvim/pack/github/start/copilot.vim
```